pyproject.toml

```diff
@@ -36,12 +36,8 @@ module-name = "glean._uniffi"
 bindings = "uniffi"
 manifest-path = "glean-core/bundle/Cargo.toml"
 
-[tool.flake8]
-max-line-length = 100
-exclude = ["glean-core/python/glean/_uniffi/glean.py"]
-ignore = [
-    "W503" # line break before binary operator - black formats it that way
-]
```

Comment:
Why aren't we migrating `max-line-length` and `ignore`?

Reply:
line-length I just forgot.
ignore because now the same tool lints and formats and so there's no disagreement about which way is right.